helm-chart/images/binderhub/requirements.txt

```diff
@@ -24,12 +24,8 @@ charset-normalizer==2.0.6
     # via requests
 cryptography==35.0.0
     # via pyopenssl
-docker==5.0.3
```

Comment:
This change seems strange, docker is still required
https://github.com/jupyterhub/binderhub/blob/2a7904dbd52cd5482ae6878cfe4f642a20ad9f77/requirements.txt#L8

Reply:
dependabot doesn't know about our custom dependency management, which merges both this requirements.in and the top-level requirements.txt

Maybe we can add a reference to `../../../requirements.txt` here and it will figure it out?

Reply:
#1433 removes the `dependencies` script in favor of calling pip-compile directly